core/src/main/java/io/confluent/kafka/schemaregistry/storage/KafkaStoreReaderThread.java

```diff
@@ -82,54 +77,58 @@ public KafkaStoreReaderThread(ZkUtils zkUtils,
     this.commitInterval = commitInterval;
     this.noopKey = noopKey;
 
-    offsetInSchemasTopic = offsetOfLastConsumedMessage(zkUtils, groupId, topic);
-    log.info("Initialized the consumer offset to " + offsetInSchemasTopic);
     Properties consumerProps = new Properties();
     consumerProps.put("group.id", this.groupId);
     consumerProps.put("client.id", "KafkaStore-reader-" + this.topic);
-    consumerProps.put("zookeeper.connect", kafkaClusterZkUrl);
-    consumerProps.put("auto.offset.reset", "smallest");
+
+    consumerProps.put("bootstrap.servers", bootstrapBrokers);
```

Comment:
Would it make sense to import `ConsumerConfig` and use for example `BOOTSTRAP_SERVERS_CONFIG` instead of `"boostrap.servers"`. 

Similar comment for other configs.